### PR TITLE
Fixes patch for ubuntu and non-working centos-8 mirrors

### PR DESCRIPTION
**Pull request by gurkanindibay**
While working on beta support for packages, I found out that docker build for ubuntu focal failed at the line below
https://github.com/citusdata/packaging/blob/351dd53fc1fd33f00af07e1b3af1e836914754f8/dockerfiles/ubuntu-focal-all/Dockerfile#L89
Additionally, centos repository mirrors were not working and I changed it with vault.epel.cloud
Removes nightlypg param from pgxn image which causes error after removal of releasepg and nightlypg parameters from pkgvars of pgxn
https://github.com/citusdata/packaging/blob/pgxn-citus/pkgvars

### CODE REVIEW

File: templates/Dockerfile-rpm.tmpl [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # vim:set ft=dockerfile:
 FROM %%os%%:%%release%%
 
+RUN [[ %%os%% != centos ]] || [[ %%release%% != 8 ]] || ( \
+    cd /etc/yum.repos.d/ && sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-* \
+    && sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.epel.cloud|g' /etc/yum.repos.d/CentOS-* \
+    )
+
```
Comment (hanefi) [inline templates/Dockerfile-rpm.tmpl:8]: The current title of the PR and the change here seems unrelated. Can you elaborate?

You can consider moving this to another PR or updating the PR title, and description

File: scripts/fetch_and_build_pgxn [hunk 2]
```diff
@@ -44,7 +44,6 @@ source /buildfiles/pkgvars
 # set default values for certain packaging variables
 declare pkglatest # to make shellcheck happy
 hubproj="${hubproj:-${pkgname}}"
-nightlypg="${nightlypg:-${releasepg}}"
```
Comment (hanefi) [inline scripts/fetch_and_build_pgxn:47]: The current title of the PR and the change here seems unrelated. Can you elaborate?

You can consider moving this to another PR or updating the PR title, and description

**Review by hanefi — CHANGES_REQUESTED** [review]

File: scripts/fetch_and_build_pgxn (hunk 2 above)
Comment (gurkanindibay) [reply scripts/fetch_and_build_pgxn:47]: Thanks for your comments. I changed the title and description. Could you check it again?

Comment (hanefi) [reply scripts/fetch_and_build_pgxn:47]: title remains unchanged
description still does not cover why `nightlypg` was removed from the files here.

Comment (gurkanindibay) [reply scripts/fetch_and_build_pgxn:47]: Apperantly, I didn't save the header :(. I changed it again.
Sorry for the loss of header

**Review by hanefi — APPROVED** [review]
